elfi/methods/methods.py

```diff
@@ -618,14 +618,14 @@ def _report_batch(self, batch_index, params, distances):
         logger.debug(str)
 
 
-class BOLFI(InferenceMethod):
+class BOLFI(BayesianOptimization):
```

Comment:
There is a test for BOLFI in `test_inference.py`. That should be updated now as this is implemented (it's currently skipped).